tooling/nargo_cli/tests/execution_success/slices_mutable_return_error/src/main.nr

```diff
@@ -0,0 +1,6 @@
+// Issue 2911
+fn main() {
+    let slice : &mut [Field] = &mut [];
```

Comment:
```suggestion
    let mut slice : &mut [Field] = &mut [];
```

Reply:
I meant to put this in execution_failure and not execution_success since this PR adds the error check for `let slice = &mut ...; slice = ...;`

Reply:
Oh, I missed that you added this test, I'll remove it since I already added `execution_failure/mutability_regression_3008`. I'll rename that one to the issue number rather than this PR number though.